### ISSUE TITLE
由onnx转换为paddle模型时报错，388-ERROR: x2paddle threw an exception

### ISSUE THREAD

**Issue by ojzhang** [PaddlePaddle/X2Paddle#960]
## 问题描述
2023-03-28 19:29:10,388-ERROR: x2paddle threw an exception, you can ask for help at: https://github.com/PaddlePaddle/X2Paddle/issues
## 具体信息
2023-03-28 19:29:10,266-INFO: Now translating model from onnx to paddle.
model ir_version: 7, op version: 12
shape inferencing ...
unsupported broadcast between 4 Reshape212_o0__d1
Stopping at incomplete shape inference at MatMul: x2paddle_attn
node inputs:
name: "x2paddle_onnx__MatMul_500"
type {
  tensor_type {
    elem_type: 1
    shape {
      dim {
        dim_value: 4
      }
      dim {
        dim_value: 1600
      }
      dim {
        dim_value: 32
      }
    }
  }
}

name: "x2paddle_onnx__MatMul_501"
type {
  tensor_type {
    elem_type: 1
    shape {
      dim {
        dim_param: "Reshape212_o0__d1"
      }
      dim {
        dim_param: "Reshape212_o0__d2"
      }
      dim {
        dim_param: "Reshape212_o0__d0"
      }
    }
  }
}

node outputs:
name: "x2paddle_attn"
type {
  tensor_type {
    elem_type: 1
    shape {
      dim {
      }
      dim {
        dim_value: 1600
      }
      dim {
        dim_param: "Reshape212_o0__d0"
      }
    }
  }
}

!!!!!!!!!!
shape inferenced.
Now, onnx2paddle support convert onnx model opset_verison [7, 8, 9, 10, 11, 12, 13, 14, 15], opset_verison of your onnx model is 12.
Total nodes: 389
Nodes converting ...
Converting node 499 ...     2023-03-28 19:29:10,388-WARNING: convert failed node:onnx__Concat_550, op_type is Resize
2023-03-28 19:29:10,388-ERROR: x2paddle threw an exception, you can ask for help at: https://github.com/PaddlePaddle/X2Paddle/issues
- 转换模型后用处
    进行模型压缩
- 模型类型
    目标检测
- 模型来源
    个人通过在yolov5加入CA注意力机制和深度可分离卷积ghostnet得到一个神经网络，用这个网络对个人标注的图像进行训练得到的模型
- 版本信息
paddlepaddle-gpu => 0.0.0.post116
X2Paddle => 1.3.9
来源框架版本(PyTorch/TF/ONNX/Caffe) => pytorch 1.12.1
- 您的联系方式(邮箱/微信/电话)
1716436594@qq.com
- onnx模型
链接：https://pan.baidu.com/s/16Pb2tktjrX8qVcj3Rnntfg 
提取码：lmlz

**Comment by wjj19950828** [comment]
@ojzhang 目前我尝试了下，是可以转的哈，安装下onnxsim简化下onnx模型

```shell
onnxsim model.onnx model_sim.onnx
x2paddle -f onnx -m model_sim.onnx -s pd_model
```

另外，可以描述一下以下两个问题吗：
1、具体业务场景
2、为什么有转到Paddle部署的需求呢？

感谢～

**Comment by xiyangyang99** [comment]
https://github.com/PaddlePaddle/X2Paddle/issues/960#issuecomment-1504536746
在大规模部署轻量模型，比如yolo系列的小模型精度无法满足场景的时候，就需要模型剪枝工具，而模型剪枝工具对比其他开源工具，paddle更为完善，paddle剪枝完毕之后，再转为onnx模型，然后trt加速部署。

**Comment by ojzhang** [comment]
@wjj19950828 主要是用paddleslim进行剪枝操作获得更轻量化的模型，用以部署在边缘设备上时获得较快的速度，满足在特定视角下快速识别芒果的需求。主要还是转为onnx模型，然后再进行部署。

**Comment by ojzhang** [comment]
@QzYER 感谢您的回复，可以用paddle进行手动剪枝吗？我看的操作是自动剪枝，需要先转为onnx模型，然后才能进行剪枝操作。

**Comment by xiyangyang99** [comment]
https://github.com/PaddlePaddle/X2Paddle/issues/960#issuecomment-1508144544
我今天已经用自己的数据集在官方的yolov5l6的权重上试过了，训练完毕之后用官方的export导出的onnx文件，然后再用paddleslim的自动化工具进行处理，最后得到quant_model.onnx可视化网络结构对比之后，发现并没有压缩，结构基本保持一致。这是自动化剪枝的结果。

手动剪枝正在尝试，还有就是那个config文件的描述好像并没有描述得太多。

**Comment by ojzhang** [comment]
@QzYER 十分感谢您的回复，我还有些问题，paddle的这个自动化剪枝是不是只能对未改动的yolo网络模型进行剪枝啊？改动过的yolov5模型是不是不能用paddle剪枝啊？比如这个 arch: YOLOv5 属性是不是只表示只能用yolov5原网络结构？
config文件如下：
Distillation:
	 alpha: 1.0
	 loss: soft_label
 Global:
	 arch: YOLOv5
	 coco_dataset_dir: dataset/coco/
	 coco_train_anno_path: annotations/instances_train2017.json
	 coco_train_image_dir: train2017
	 coco_val_anno_path: annotations/instances_val2017.json
	 coco_val_image_dir: val2017
	 image_path: None
	 model_dir: best_yolov5s.onnx
 QuantAware:
	 activation_quantize_type: moving_average_abs_max
	 onnx_format: True
	 quantize_op_types: ['conv2d', 'depthwise_conv2d']
	 use_pact: True
 TrainConfig:
	 eval_iter: 1000
	 learning_rate: 1e-05
	 optimizer_builder:
		 optimizer:
			 type: SGD
		 weight_decay: 4e-05
	 target_metric: 0.365
	 train_iter: 3000

**Comment by ojzhang** [comment]
@QzYER 十分感谢您的回复，我还有些问题，paddle的这个自动化剪枝是不是只能对未改动的yolo网络模型进行剪枝啊？改动过的yolov5模型是不是不能用paddle剪枝啊？比如这个 arch: YOLOv5 属性是不是只表示只能用yolov5原网络结构？
config文件如下：
Distillation:
  alpha: 1.0
  loss: soft_label
Global:
  arch: YOLOv5
  coco_dataset_dir: dataset/coco/
  coco_train_anno_path: annotations/instances_train2017.json
  coco_train_image_dir: train2017
  coco_val_anno_path: annotations/instances_val2017.json
  coco_val_image_dir: val2017
  image_path: None
  model_dir: best_yolov5s.onnx
QuantAware:
  activation_quantize_type: moving_average_abs_max
  onnx_format: True
  quantize_op_types: ['conv2d', 'depthwise_conv2d']
  use_pact: True
TrainConfig:
  eval_iter: 1000
  learning_rate: 1e-05
  optimizer_builder:
    optimizer:
      type: SGD
    weight_decay: 4e-05
  target_metric: 0.365
  train_iter: 3000

**Comment by ojzhang** [comment]
@QzYER 十分感谢您的回复，我还有些问题，paddle的这个自动化剪枝是不是只能对未改动的yolo网络模型进行剪枝啊？改动过的yolov5模型是不是不能用paddle剪枝啊？比如这个 arch: YOLOv5 属性是不是只表示只能用yolov5原网络结构？
config文件如下：
```
Distillation:
  alpha: 1.0
  loss: soft_label
Global:
  arch: YOLOv5
  coco_dataset_dir: dataset/coco/
  coco_train_anno_path: annotations/instances_train2017.json
  coco_train_image_dir: train2017
  coco_val_anno_path: annotations/instances_val2017.json
  coco_val_image_dir: val2017
  image_path: None
  model_dir: best_yolov5s.onnx
QuantAware:
  activation_quantize_type: moving_average_abs_max
  onnx_format: True
  quantize_op_types: ['conv2d', 'depthwise_conv2d']
  use_pact: True
TrainConfig:
  eval_iter: 1000
  learning_rate: 1e-05
  optimizer_builder:
    optimizer:
      type: SGD
    weight_decay: 4e-05
  target_metric: 0.365
  train_iter: 3000
```
